runtime/compiler/runtime/RelocationRuntimeLogger.cpp

```diff
@@ -143,33 +142,33 @@ TR_RelocationRuntimeLogger::method(bool newLine)
    int signatureLen= J9UTF8_LENGTH(J9ROMMETHOD_SIGNATURE(J9_ROM_METHOD_FROM_RAM_METHOD(reloRuntime()->method())));
 
    bool wasLocked = lockLog();
-   JITRT_PRINTF(jitConfig())(jitConfig(), patternString, classNameLen, className,
-                                                         methodNameLen, methodName,
-                                                         signatureLen, signature);
+   rtlogPrintf(jitConfig(), reloRuntime()->fej9()->_compInfoPT, (const char *)patternString,
+               classNameLen, className, methodNameLen, methodName, signatureLen, signature);
    unlockLog(wasLocked);
    }
 
 void
 TR_RelocationRuntimeLogger::exceptionTable()
    {
    J9JITExceptionTable *data = reloRuntime()->exceptionTable();
+   TR::CompilationInfoPerThread *compInfoPT = reloRuntime()->fej9()->_compInfoPT;
 
    bool wasLocked = lockLog();
 
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12s",   "startPC");
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12s",   "endPC");
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-8s",    "size");
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-14s",   "gcStackAtlas");
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12s\n", "bodyInfo");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12s",   "startPC");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12s",   "endPC");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-8s",    "size");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-14s",   "gcStackAtlas");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12s\n", "bodyInfo");
 
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12p",   data->startPC);
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12p",   data->endPC);
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-8x",    data->size);
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-14p",   data->gcStackAtlas);
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12p\n", data->bodyInfo);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12p",   data->startPC);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12p",   data->endPC);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-8x",    data->size);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-14p",   data->gcStackAtlas);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12p\n", data->bodyInfo);
 
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12s\n", "inlinedCalls");
-   JITRT_PRINTF(jitConfig())(jitConfig(), "%-12x\n", data->inlinedCalls);
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12s\n", "inlinedCalls");
+   rtlogPrintf(jitConfig(), compInfoPT, "%-12x\n", data->inlinedCalls);
```

Comment:
A couple more preexisting little things here:
- `startPC` and `endPC` are `UDATA`. I think if we're using `%p` for them, we should explicitly cast them to pointers, e.g. `void*`. Either that or change `"%-12p"` to `"%-12" OMR_PRIxPTR`
- `inlinedCalls` is a pointer, so I think that should use `%p` rather than `%x`

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/pull/14210/commits/de354ea9c291db3231788240d621ed75f430322c